code/game/objects/items/weapons/autolathe_disks.dm

```diff
@@ -1188,22 +1188,77 @@
 	)
 
 /obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee
-	disk_name = "NeoTheology Armory - Melee Weapons"
+	disk_name = "NeoTheology Armory - Basic Melee Weapons"
 	icon_state = "neotheology"
 	spawn_blacklisted = TRUE
 	rarity_value = 50
 	license = -1
 	designs = list(
 		/datum/design/autolathe/sword/nt_sword,
-		/datum/design/autolathe/sword/nt_longsword,
 		/datum/design/autolathe/sword/nt_dagger,
-		/datum/design/autolathe/sword/nt_halberd,
+		/datum/design/bioprinter/storage/sheath,
+		/datum/design/autolathe/tool_upgrade/sanctifier
+	)
+
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/nt_melee/longsword
+	disk_name = "NeoTheology Armory - NT Longsword"
+	icon_state = "neotheology"
+	spawn_blacklisted = TRUE
+	rarity_value = 50
```

Comment:
While we're at it, let's go ahead and make these disks valuable as well. After all, they contain designs that are intended to only be known by NeoTheologian artisans - as such, they go for a very high price on the export market.

Go ahead and make all of these disks sell for 1000 credits on the market.

Reply:
Sure.

Reply:
Done.